### PR TITLE
Ninth World Guidebook

### PR DESCRIPTION
**Pull request by shaydin**
All Descriptors and Foci from the NWG.  There are no additional Types from this source book.

### CODE REVIEW

**Comment by darkliquid** [comment]
Awesome, thank you so much! I'll write the extra bits of glue code to pull this into the generator and rebuild soon!

File: src/data/chargen/books/ninth-world-guidebook [hunk 1]
```diff
@@ -0,0 +1,227 @@
+var descriptors = [
+  {
+    name: 'Coraoan',
+    sourcebook: 'Ninth World Guidebook',
+    page: 215,
+    sources: [
+      { sourcebook: 'Ninth World Guidebook', page: 215 }
```
Comment (darkliquid) [inline src/data/chargen/books/ninth-world-guidebook]: I can't remember off the top of my head, but I *think* this isn't necessary when it's a duplicate of the main sourcebook/page.

File: src/data/chargen/books/ninth-world-guidebook [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+    page: 216,
+    sources: [
+      { sourcebook: 'Ninth World Guidebook', page: 216 }
+    ],
+    stats: {
+      intellect: 3
+    },
+    shins: 10,
+    extras: [
+      'Steadfast: Your faith gives you the foundation to deal with stress. When something would alter the difficulty of your action to your detriment, you can negate one step of the modification. Once you use this ability, you cannot use it again until after your next ten-hour recovery roll.'
```
Comment (darkliquid) [inline src/data/chargen/books/ninth-world-guidebook]: I'm concerned this exceeds the limits of what MCG allow for generators, as it contains basically the entire descriptive text of it. Stripping it back to a one sentence summary would probably be advised.

File: src/data/chargen/books/ninth-world-guidebook [hunk 3]
```diff
@@ -0,0 +1,227 @@
...
+    skills: {
+      trained: [
+        'All tasks involving positive social interactions'
+      ],
+      inability: [
+        ' If you go more than 28 hours without switching between water breathing and air breathing, you gain an inability in movement-related tasks, including running, jumping, and climbing. The inability persists until you make the switch between environments for at least a few minutes.'
+      ]
+    },
+    extras: [
+      'Gregarious: Despite your unusual appearance, your social skills charm even the most standoffish of creatures. You are trained in all tasks involving positive social interactions.',
```
Comment (darkliquid) [inline src/data/chargen/books/ninth-world-guidebook]: I'm concerned these exceed the limits of what MCG allow for generators, as it contains basically the entire descriptive text. Stripping them back to a one sentence summary would probably be advised.

File: src/data/chargen/books/ninth-world-guidebook [hunk 4]
```diff
@@ -0,0 +1,227 @@
...
+      intellect: 4
+    },
+    shins: 10,
+    skills: {
+      trained: [
+        'Tasks involving botany, zoology, or biology',
+        'Climbing'
+      ],
+      inability: [
+        'Understanding or using powered numenera, including ray projectors, automatons, force fields, vehicles, and so on—anything requiring a nonliving power source'
```
Comment (darkliquid) [inline src/data/chargen/books/ninth-world-guidebook]: Can probably summarise this more concisely.

File: src/data/chargen/books/ninth-world-guidebook [hunk 5]
```diff
@@ -0,0 +1,227 @@
...
+    page: 219,
+    sources: [
+      { sourcebook: 'Ninth World Guidebook', page: 219 }
+    ],
+    stats: {
+      speed: 4
+    },
+    shins: 10,
+    extras: [
+      'Pliant: You are at home on both water and land, adept at switching between the two. At the beginning of each day, you can choose to be trained either in running and jumping, or in swimming and sailing'
```
Comment (darkliquid) [inline src/data/chargen/books/ninth-world-guidebook]: Can probably be more concise here.

**Comment by shaydin** [comment]
Sounds good, all completely understandable.  Let me know how the next iteration looks.

**Comment by darkliquid** [comment]
Weirdly you now have 2 files. One with a .js extension and one without.

**Comment by darkliquid** [comment]
Okay, I ended up manually merging it on my computer as I wanted to make some extra changes and noticed a couple of things that needed fixing. You can take a look at the commit here: https://github.com/darkliquid/numenera-tools/commit/86d0749ad5460c972b4eda617b9be2b484ec60fb

Thanks so much for lending a helping hand!

**Pull request closed without merging**